tests/test_po.py

```diff
@@ -56,15 +58,15 @@ class TestGTranslator(unittest.TestCase):
     def test_gtranslator(self):
         """Test all pofiles for GTranslator brokenness."""
         for f in get_pofiles():
-            with open(f, "rb") as fd:
+            with open(f) as fd:
```

Comment:
`encoding="UTF-8"`?  All of the .po files in the source tree use UTF-8.

Reply:
Thanks, I just assume everyone must be on UTF-8 in 2002... Done in a new commit.